assets/css/main.css

```diff
@@ -335,9 +335,7 @@ a {
 #hero-index {
   background: linear-gradient(rgba(var(--primary-dark), 0.7), rgba(var(--primary-dark), 0.7)),
     url('/assets/images/hero-index.jpg');
-  background-position: top;
-  background-repeat: no-repeat;
-  background-size: cover;
+  background-position: 53% 0%;
```

Comment:
These percents are a codesmell, can you lock them at a specific pixel?

Reply:
Done.